### PR TITLE
Adding in the SES module

### PR DESCRIPTION
**Pull request by goldblatt**
### Description
Set up SES via terraform and pass the sender email to the app. 

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
Fixes #<issue_number>; Fixes #<issue_number>; Fixes #<issue_number>...

### CODE REVIEW

File: cloud/aws/providers.tf [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
+      version = "3.72.0"
+    }
+  }
+} 
+
+provider "aws" {
+  region = "us-east-1"
```
Comment (bion) [inline cloud/aws/providers.tf:11]: suggestion (non-blocking, if-minor): use a variable

File: cloud/deploys/staging/providers.tf [hunk 2]
```diff
@@ -1,3 +1,7 @@
 provider "azurerm" {
   features {}
 }
+
+provider "aws" {
+  region = "us-east-1"
```
Comment (bion) [inline cloud/deploys/staging/providers.tf:6]: suggestion (non-blocking, if-minor): use a variable

File: cloud/deploys/staging/main.tf [hunk 3]
```diff
@@ -21,4 +25,10 @@ module "app" {
 
   application_name = var.application_name
   app_secret_key   = var.app_secret_key
+  ses_sender_email = var.sender_email_address
+}
+
+module "email_service" {
+  source               = "../../aws/modules/ses"
+  sender_email_address = var.sender_email_address
```
Comment (bion) [inline cloud/deploys/staging/main.tf:33]: suggestion (non-blocking, if-minor): could we make using SES with Azure optional on a config value? very much an only-if-minor suggestion

**Review by bion — APPROVED** [review]

File: cloud/deploys/staging/main.tf (hunk 3 above)
Comment (goldblatt) [reply cloud/deploys/staging/main.tf:33]: i would think that this file would change if we were going to the deploy differently if that makes sense not passing in the config values here but fully just not having this exists in the deploys folder

**Comment by bion** [comment]
aaaah that makes a lot of sense

**Pull request merged**